src/libDirectoryService/DSBlockPreProcessing.cpp

```diff
@@ -209,10 +212,15 @@ void DirectoryService::ComputeSharding(const VectorOfPoWSoln& sortedPoWSolns) {
   }
 }
 
-void DirectoryService::InjectPoWForDSNode(VectorOfPoWSoln& sortedPoWSolns,
-                                          unsigned int numOfProposedDSMembers) {
-  // Add the oldest n DS committee member to m_allPoWs and m_allPoWConns so it
-  // gets included in sharding structure
+void DirectoryService::InjectPoWForDSNode(
+    VectorOfPoWSoln& sortedPoWSolns, unsigned int numOfProposedDSMembers,
+    std::vector<PubKey>& removeDSNodePubkeys) {
```

Comment:
use const infront of std::vector<PubKey>&

Reply:
Thanks, made the change.